src/cluster-configuration/deploy/gpu-configuration/gpu-configuration.json.template

```diff
@@ -2,8 +2,9 @@
   "nodes" : {
 {% for host in machinelist if 'gpu' in machineinfo[ machinelist[ host ][ 'machinetype' ] ] %}
     "{{machinelist[ host ]['nodename']}}" : {
-        "gpuType" : "{{ machineinfo[ machinelist[ host ][ 'machinetype' ] ]['gpu']['type'] }}"
+        "gpuType" : "{{ machineinfo[ machinelist[ host ][ 'machinetype' ] ]['gpu']['type'] }}",
```

Comment:
will this config let deployer hands on config it or generated by script?

Reply:
generated by script, This is cluster-configuration's template, will expand on start of cluster-configuration service which node-exporter depended on.

Reply:
ok, so if contains 2 gpu server with 2 types of gpu number and type, it will detected by scripts ?

Reply:
They will be copied from `cluster-configuration.yaml` not detected by anything.

Reply:
if it is not need deployer hands on config, it is ok.